benchmarks/views/user.py

```
@@ -282,3 +320,35 @@ def post(self, request):
         model.public = public
         model.save(update_fields=['public'])
         return JsonResponse("success", safe=False)
+
+
+def get_secret(secret_name, region_name='us-east-2'):
+    session = boto3.session.Session()
+    _logger.info("Fetch secret from secret manager")
+    try:
+        client = session.client(
+            service_name='secretsmanager',
+            region_name=region_name,
+        )
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'ResourceNotFoundException':
+            _logger.error("The requested secret " + secret_name + " was not found")
+        elif e.response['Error']['Code'] == 'InvalidRequestException':
+            _logger.error("The request was invalid due to:", e)
+        elif e.response['Error']['Code'] == 'InvalidParameterException':
+            _logger.error("The request had invalid params:", e)
+    except Exception as e:
+        _logger.error("The request failed with:", e)
```

don't we want to `raise e` again instead of just logging?